BankSDK/GiniBankSDKExample/GiniBankSDKExample/SettingsViewController.swift

```diff
@@ -1,115 +1,223 @@
 //
 //  SettingsViewController.swift
-//  Example Swift
+//  GiniBankSDKExample
 //
-//  Created by Nadya Karaban on 19.02.21.
+//  Created by Valentina Iancu on 07.06.23.
 //
 
-import Foundation
 import UIKit
 import GiniCaptureSDK
 import AVFoundation
 
 protocol SettingsViewControllerDelegate: AnyObject {
-    func settings(settingViewController: SettingsViewController,
-                  didChangeConfiguration captureConfiguration: GiniConfiguration)
+	func settings(settingViewController: SettingsViewController,
+				  didChangeConfiguration captureConfiguration: GiniConfiguration)
 }
 
-final class SettingsViewController: UIViewController {
-    
-    weak var delegate: SettingsViewControllerDelegate?
-    var giniConfiguration: GiniConfiguration!
+class SettingsViewController: UIViewController {
```

Comment:
[Clarification] why  SettingsViewController is not `final` anymore?

Reply:
I forgot to put it back since I created a new file in the beginning and after that rename it to SettingsViewController.
I will fix it :D.
Thanks for catching it!